ethereum/core/src/main/java/org/hyperledger/besu/ethereum/core/Transaction.java

```diff
@@ -38,16 +39,16 @@
 
   // Used for transactions that are not tied to a specific chain
   // (e.g. does not have a chain id associated with it).
-  private static final BigInteger REPLAY_UNPROTECTED_V_BASE = BigInteger.valueOf(27);
-  private static final BigInteger REPLAY_UNPROTECTED_V_BASE_PLUS_1 = BigInteger.valueOf(28);
+  public static final BigInteger REPLAY_UNPROTECTED_V_BASE = BigInteger.valueOf(27);
+  public static final BigInteger REPLAY_UNPROTECTED_V_BASE_PLUS_1 = BigInteger.valueOf(28);
 
-  private static final BigInteger REPLAY_PROTECTED_V_BASE = BigInteger.valueOf(35);
+  public static final BigInteger REPLAY_PROTECTED_V_BASE = BigInteger.valueOf(35);
 
   // The v signature parameter starts at 36 because 1 is the first valid chainId so:
   // chainId > 1 implies that 2 * chainId + V_BASE > 36.
-  private static final BigInteger REPLAY_PROTECTED_V_MIN = BigInteger.valueOf(36);
+  public static final BigInteger REPLAY_PROTECTED_V_MIN = BigInteger.valueOf(36);
 
-  private static final BigInteger TWO = BigInteger.valueOf(2);
+  public static final BigInteger TWO = BigInteger.valueOf(2);
```

Comment:
I'm wondering if there is a better name for this

Reply:
Hmm. No better name that what the constant is really, and it is literally the number two.